source/process/training.rst

```diff
@@ -106,7 +106,66 @@ Mattermost Staff Meetings
 
 For confidentiality of customers, internal process meetings are limited to Mattermost staff only.
 
+Customer Obsession All-Hands Meeting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Attendees:
+
+- All Mattermost Staff
+
+Objectives:
+
+- Share important company-wide updates with staff across the organization.
+- Educate staff on key customer use cases and feature requests.
```

Comment:
Propose instead of "feature requests" we call them "needs", it's not about customers filing issues, it's about us understanding what they need whether or not they ask for it.

Reply:
Updated